.github/scripts/get_top_block.py

```diff
@@ -1,6 +1,19 @@
 #!/usr/bin/env python3
+import sys
 from sys import argv
+import time
+
 from substrateinterface import SubstrateInterface
-chain = SubstrateInterface(url=argv[1])
-number = chain.get_block()['header']['number']
-print(number)
+connection_attempts = 0
+
+while connection_attempts < 30:
+    try:
+        chain = SubstrateInterface(url=argv[1])
+        number = chain.get_block()['header']['number']
+        print(number)
+        sys.exit(0)
+    except ConnectionRefusedError:
```

Comment:
if you want to make it more robust go for:
`except Exception as e:`
then it catches (almost) everything

Reply:
This is now done